src/HornedBeasts.js

```diff
@@ -0,0 +1,17 @@
+import { Component } from 'react';
+
+class HornedBeasts extends Component {
+    render() {
+        return (
+        <div>
+        <h2>{this.props.name}</h2>
+        <img src={this.props.imageUrl} alt={this.props.description} title={this.props.name}></img> 
+        <p>Description: {this.props.description}</p>
+
```

Comment:
Really good work so far 👍 

For these lines we can indent (not sure if you are a 2 space person or 4 :)). No extra whitespace needed after the `<p>` tag.